internal/service/privatelinkendpointservicedatafederationonlinearchive/data_source_privatelink_endpoint_service_data_federation_online_archive.go

```diff
@@ -38,24 +38,24 @@ func DataSource() *schema.Resource {
 }
 
 func dataSourceMongoDBAtlasPrivatelinkEndpointServiceDataFederationOnlineArchiveRead(ctx context.Context, d *schema.ResourceData, meta any) diag.Diagnostics {
```

Comment:
recommend to change algo function names, e.g. dataSourceMongoDBAtlasPrivatelinkEndpointServiceDataFederationOnlineArchiveRead to dataSourceMongoRead